src/mainwindow.h

```diff
@@ -81,12 +81,12 @@ class MainWindow : public QLabel
     void updateZoom();
 
 public slots:
-    void fullScreenshot();
+    void fullScreenshot(QString savePath="");
```

Comment:
I think we don't need this extra parameter, you can just use `m_saveFilename`?

Reply:
thanks for the great suggestion